### ISSUE TITLE
Conclusion of 4.118's proof unclear, potentially with issues

### ISSUE THREAD

**Issue by Antymon** [mml-book/mml-book.github.io#123]
**Describe the mistake**
I find the proof's conclusion of the latter part (4.118) of Eckart-Young Theorem very hard to follow. There may be some mistakes, but I understand too little to say for sure. I summed up my struggle line by line:

Line 2515: 

V appears out of sudden after not being mentioned for a longer while.
Most natural would be to think of it as a right singular matrix, but it's clearly stated that V represents a subspace.

1) Is V a domain?
2) Does it relate to nullspace Z?
3) How to explain its dimensions (n-k)?

Line 2516: 

4) "There is" means "exists"?
5) What does "that" subspace represent, codomain?
6) Why is it n+1 dimensional? Just as a guess - shouldn't it be at least k+1 dimensional?

Line 2517:

7) Shouldn't a subspace spanned by a single vector have just 1 dimension? If so, there is something wrong at this line.

Further lines:

Due to confusion I struggle to identify entities related to Null-Rank Theorem.

As a side note, since ||Ax||_2 >= sigma_(k+1)||x||_2 is claimed, shouldn't that suffice to show contradiction even without referring to Null-Rank Theorem? However I don't know why this would hold either. That is, why length of Ax for any x would be >= than lengthening ||x||_2 by singular value k+1 where k<=rank(A)?

**Location**
Please provide the 
1. version 2018-09-21
2. Chapter 4
3. page 129

**Comment by AnalogAldo** [comment]
ok, this has now been rephrased after off-line discussion with you.
